generators/chipyard/src/main/scala/config/BoomConfigs.scala

```diff
@@ -3,60 +3,116 @@ package chipyard
 import org.chipsalliance.cde.config.{Config}
 
 // ---------------------
-// BOOM Configs
+// BOOM V3 Configs
+// Performant, stable baseline
 // ---------------------
 
-class SmallBoomConfig extends Config(
-  new boom.common.WithNSmallBooms(1) ++                          // small boom config
+class SmallBoomV3Config extends Config(
```

Comment:
This will break FireSim's default configs: https://github.com/firesim/firesim/blob/main/deploy/sample-backup-configs/sample_config_build_recipes.yaml. Can you bump FireSim as well?

Reply:
I'll revert the name

Reply:
Nvm I see the problem....